base/src/main/java/io/spine/protobuf/Durations2.java

```diff
@@ -185,6 +189,7 @@ public static Duration hoursAndMinutes(long hours, long minutes) {
      * @deprecated please use {@link Durations#toNanos(Duration)}.
      */
     @Deprecated
+    @InlineMe(replacement = "Durations.toNanos(duration)", imports = PROTOBUF_DURATIONS)
     public static long toNanos(Duration duration) {
```

Comment:
Do we still need these methods? Maybe we could just kill them off?

It's supposed to happen some time before the 2.0 release anyway, in my opinion.